lib/pages/home.dart

```diff
@@ -414,27 +415,35 @@ class HomePageState extends State<HomePage>
       // Start fetch
       setState(() => _isFetching = true);
       try {
-        _currentFetchPlanOperation = car
-            ? requestManagerBloc.fetchCarPlan(
-                context,
-                _data.fromPlace,
-                _data.toPlace,
-              )
-            : requestManagerBloc.fetchTransitPlan(
-                context,
-                _data.fromPlace,
-                _data.toPlace,
-              );
+        if (car) {
+          _currentFetchPlanOperation = requestManagerBloc.fetchCarPlan(
+            context,
+            _data.fromPlace,
+            _data.toPlace,
+          );
+        } else if (features.useBikeRoutes) {
+          _currentFetchPlanOperation = requestManagerBloc.fetchBikePlan(
+            context,
+            _data.fromPlace,
+            _data.toPlace,
+          );
+        } else {
+          _currentFetchPlanOperation = requestManagerBloc.fetchTransitPlan(
+            context,
+            _data.fromPlace,
+            _data.toPlace,
+          );
+        }
 
         Plan plan = await _currentFetchPlanOperation.valueOrCancellation(null);
 
         if (plan == null) {
           throw "Canceled by user";
```

Comment:
This is not part of this PR, but we should create a fitting Error class for something like that in the future.